xonsh/bash_completion.py

```diff
@@ -0,0 +1,276 @@
+import os
+import re
+import sys
+import platform
+import shlex
+import pathlib
+import subprocess
+
+
```

Comment:
Another thing I have to be nice is to have `__version__ = 'x.y.z'` in these files, so you know if you copy is out of date.